[ig/security] scope: change "incubates" to identifies, "initiating work" to tests

tantek opened the pull request:
As the existing Out of Scope section says, the IG must hand over standards work or incubation to other groups, thus for consistency the scope section should drop incubation (suggested text replacement with the more abstract "exploration") and focus on security issues on standards work rather than standards work itself. Also replaced "initiating the work" with a more limited suggested phrase of "developing tests".

IGs must not explicitly "incubate", as in the context of W3C, that has technical standards work implications which only CGs and WGs are properly setup (IPR) to do.

cc: @jyasskin @cwilso

tantek committed:
scope: replace out-of-scope "incubates" with explores, initiating work to tests

As the existing Out of Scope section says, the IG must hand over standards work or incubation to other groups, thus for consistency the scope section should drop incubation (suggested text replacement with the more abstract "exploration") and focus on security issues on standards work rather than standards work itself. Replaced "initiating the work" with a more limited suggested phrase of "developing tests"

simoneonofri reviewed (approved):
Hi @tantek, yes the wording “exploring” is more suitable. As I also specify in the out-of-scope, the issue is not incubating standards but ideas that can then be developed in *other* groups.
I put myself as a reviewer I accepted the change, is this enough for you to “solve” the FO before the WBS closes? There is a lot of work to be done on reviews and I would like to start as soon as possible :)

In parallel, I answered you for, the Process issue on the other PR. It's okay to do the revert, but I was also thinking of making the issue more generic by removing the reference to groups.

Ok for the mission and thanks for the typo!

jyasskin commented on 2024/ig-security.html:
```diff
@@ -160,7 +160,7 @@ <h2>Scope</h2>
 	<p>SING supports, promotes, and structures the threat modeling for web standards and technologies. This approach can be used, along with other groups, for threats of different types such as security, privacy, and other kinds of harm. Threat modeling is a joint activity with threat experts and groups developing technology or other documentation. It can be used to get an understanding of the impac ...
         <p>SING provides "<a href="https://www.w3.org/Guide/documentreview/">horizontal review</a>", offering groups on-request guidance on security issues and mitigations specific to their technologies. SING aims to offer this review as early in the technology development lifecycle as requested, observing that early feedback is often more helpful. SING may also seek out technologies that benefit ...
-        <p>SING incubates standards work on security issues by collecting requirements, prototyping, and/or initiating the work within the IG and recommending that the W3C move the work into other groups when appropriate.</p>
+        <p>SING explores security issues on standards work by collecting requirements, prototyping, and/or developing tests within the IG and recommending that the W3C move the work into other groups when appropriate.</p>
```
I think this was better as "explores standards work on security issues" or s/on/for/. It's the standards work that the IG will eventually propose, not the security issues.

tantek replied:
@jyasskin "explores standards work" sounds far too broad for an IG, IGs are not supposed to pursue actual standards work at all, in any way, whether incubating or exploring.

simoneonofri replied:
premised that the origin of that sentence was taken from PING, which I used as boilerplate:

> PING incubates standards work on privacy issues by collecting requirements, prototyping, and/or initiating the work within the IG and recommending that the W3C move the work into other groups when appropriate. For example, PING may incubate work to reconcile differences in browser vendors' approaches to privacy-protecting features, potentially to include guidelines, best practices, and standards work.

considering that I had put OOS the development of standards, I am prone to merge the change proposed by @tantek as already anticipated.

@jyasskin @chrisn @msporny can you provide your review/approval if it makes sense to you?

jyasskin replied:
Other IGs have:
* "[Identify and communicate standardization opportunities for W3C](https://www.w3.org/2022/11/chinese-web-ig-charter.html#scope:~:text=Identify%20and%20communicate%20standardization%20opportunities%20for%20W3C)"
* "[the primary focus of the group is expected to be on discovering and defining standardization items](https://www.w3.org/2023/12/web-networks-charter.html#scope:~:text=the%20primary%20focus%20of%20the%20group%20is%20expected%20to%20be%20on%20discovering%20and%20defining%20standardization%20items)"
* "[identifying appropriate standardization targets](https://www.w3.org/2024/04/wot-ig-2024.html#:~:text=identifying%20appropriate%20standardization%20targets)"
* "[Identify standardization opportunities](https://www.w3.org/securepay/charter-2023.html#activities:~:text=Identify%20standardization%20opportunities)"
* "[Identification and prioritization of gaps in Web standards for Smart City use cases](https://www.w3.org/2024/06/smart-cities/#scope:~:text=Identification%20and%20prioritization%20of%20gaps%20in%20Web%20standards%20for%20Smart%20City%20use%20cases)"

So ... maybe "identifies" is a better word than "explores"?

```suggestion
        <p>SING identifies standardization work on security issues by collecting requirements, prototyping, and/or developing tests within the IG and recommending that the W3C move the work into other groups when appropriate.</p>
```

I don't strongly object to @tantek's change; I'm just not sure what a "security issue on standards work" is, or how one explores a security issue by collecting requirements, etc.

tantek commented:
> I put myself as a reviewer I accepted the change, is this enough for you to “solve” the FO before the WBS closes?

That doesn't work because if the FO is removed, then there is no reason to update the charter and ask the other voters if they are ok with the update.

> There is a lot of work to be done on reviews and I would like to start as soon as possible :)

@simoneonofri, I believe the right steps are roughly:
1. The review closes
2. The Team (you in this case I presume) reviews the feedback, all suggestions. 
3. You decide which ones make sense for improving the charter (as you have), and are worth having to follow-up with who voted
4. Merge PRs for suggestions, and make your own minor edits to handle suggestions, with your editorial judgment for what specifically will reach the greatest consensus across the feedback, and produce an updated charter accordingly
5. Reach out to everyone who voted, send them the updated charter, and ask if anyone would change their vote, especially any FO(s)
6. The AC reps who voted either say nothing (no new objections), or they may raise objections to the changes, or they may indicate which suggestions have been handled by the updated charter (including any FOs)

Once you handle steps 2-5, then I (and other AC reps who voted) and respond per step 6 and lift any FOs

@plehegar can correct any mistakes I made above.

simoneonofri commented:
@tantek 

> Once you handle steps 2-5, then I (and other AC reps who voted) respond per step 6 and lift any FOs

First, thank you for your effort to improve the charter, so we are now on steps 3/4.

In any case, my approach is to consider and process feedback, even non-blocking comments. If you saw me in a hurry, I know there's much to review for security, and I'd like to provide this service to the community as soon as possible. 🙏

tantek committed:
Update 2024/ig-security.html

@jyasskin's mitigation is also acceptable. Merging. @simoneonofri, if you want to further iterate, please feel free to offer suggestions.

Co-authored-by: Jeffrey Yasskin <jyasskin@gmail.com>

simoneonofri commented:
@tantek, just to be sure, are you happy with the PR, particularly regarding the FO? thank you :)

tantek commented:
@simoneonofri yes, merging this PR does resolve the FO in Mozilla Charter poll response.

I also noticed another minor improvement opportunity for proper coordination on the ES/JS security model, which is handled by TC39-TG3 (not OpenJS Foundation).

Please see: https://github.com/w3c/charter-drafts/pull/617 which only adds TC39-TG3 as a minor change — I left the coordination with OpenJS Foundation as-is in order to make this suggested change as small as possible.

simoneonofri commented:
thank you @tantek. Approved and merged  https://github.com/w3c/charter-drafts/pull/617
